### PR TITLE
[TEST]: Regular Flow: New interaction approach: Func-tests: Eliminating flowHelper(v1/v2) usage

### PR DESCRIPTION
**Pull request by yuliiamir**
It appears that both flowHelpers and flowHelperV2 are used in performance tests. As a result, the elimination of usage of these helpers is implemented for the functional tests in the scope of this patch.  Both flowHelpers and flowHelperV2 have methods that are used ONLY by performance tests.

### CODE REVIEW

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowExtended.groovy [hunk 1]
```diff
@@ -413,6 +421,7 @@ class FlowExtended {
     }
 
     FlowReroutePayload sync() {
+        log.debug("Sync Flow '$flowId'")
```
Comment (izadorozhna) [inline src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowExtended.groovy:424]: Very minor. We are using the "present continues" -ing action for the logs, so maybe Sync -> Syncing. But this is up to you

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowSpec.groovy [hunk 2]
```diff
@@ -249,12 +251,16 @@ class VxlanFlowSpec extends HealthCheckSpecification {
                 .build().create()
 
         then: "Flow is created with protected path"
-        def flowPathInfo = flow.retrieveDetails()
-        flowPathInfo.allocateProtectedPath
+        def flowPathInfo = flow.retrieveAllEntityPaths()
```
Comment (izadorozhna) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowSpec.groovy:254]: Oh, it is insane, it was me who wrote `def flowPathInfo = flow.retrieveDetails()`. Sorry for this typo :)

**Review by izadorozhna — APPROVED** [review]

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowSpec.groovy (hunk 2 above)
Comment (yuliiamir) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowSpec.groovy:254]: np 😉

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowExtended.groovy (hunk 1 above)
Comment (yuliiamir) [reply src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowExtended.groovy:424]: sure, will update in my next PR

**Pull request merged**